logic/strategy/src/promiscuous.rs

```diff
@@ -76,6 +76,7 @@ pub struct PromiscuousStrategyConfig {
     pub enforce_max_channels: bool,
 
     /// Specifies minimum version of the peer the strategy should open a channel to.
+    /// Accepts semver syntax.
     /// Default is ">=2.0.0"
```

Comment:
Pls, have a look at the generated docs, the newlines are necessary for readability formatting.

```suggestion
    /// Specifies minimum version of the peer the strategy should open a channel to.
    ///
    /// Accepts semver syntax.
    ///
    /// Default is ">=2.0.0"
```